packages/agent/src/oidc-v2.ts

```diff
@@ -297,7 +250,7 @@ async function deriveCodeChallenge(codeVerifier: Uint8Array) {
 // TODO: when implementing pure OIDC split up the Web5 and OIDC params
```

Comment:
Sanity: Is this still applicable? Obsolete? Remove?

Reply:
Still applies

Reply:
Sorry I don't understand the TODO, can you elaborate/clarify? I thought what you are doing is ODIC compliant already? If so, what does it mean by "when implementing pure OIDC"?

Reply:
We're doing web5 connect in this PR. The comment says that when doing a PR for OIDC to add more separation of concerns between web5 connect and OIDC.

Reply:
There are other PRs that will come in the future that will refine our code for use with OIDC4VC SIOPv2 rather than only Web5 Connect. The web5 connect PR is tested on web5 connect only, it is not tested with third party OIDC4VC SIOPv2 flows. There will be more refinement to work with both protocols.